### PR TITLE
fix(lint): sanitize PR messages

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Sanitize PR title and body when doing the lint pull request stage.
Also explicitly set type-enum values so we add "example" to the list.

### CODE REVIEW

File: .github/workflows/pr-commitlint.yml [hunk 1]
```diff
@@ -18,4 +18,6 @@ jobs:
           last_commit=HEAD^2 # don't lint the merge commit
           npx commitlint --from $first_commit~1 --to $last_commit -V
       - name: Lint Pull Request
-        run: echo "${{ github.event.pull_request.title }}"$'\n\n'"${{ github.event.pull_request.body }}" | npx commitlint -V
```
Comment (Hoverbear) [inline .github/workflows/pr-commitlint.yml:21]: I **think** you could get by with

```suggestion
        run: echo '${{ github.event.pull_request.title }}\n\n${{ github.event.pull_request.body }}' | npx commitlint -V
```

File: .github/workflows/pr-commitlint.yml [hunk 2]
```diff
@@ -18,4 +18,12 @@ jobs:
...
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+          BODY: ${{ github.event.pull_request.body }}
+          NILL: $'\n'
+        run: |
+          printenv TITLE NILL BODY > a.txt
+          cat a.txt
+          npx commitlint -e a.txt -V
+          printenv TITLE NILL BODY | npx commitlint -V
```
Comment (Hoverbear) [inline .github/workflows/pr-commitlint.yml:29]: I think this is a duplicate?

Comment (tiagolobocastro) [reply .github/workflows/pr-commitlint.yml]: just me testing things, it's ready now

**Review by Hoverbear — APPROVED** [review]

**Review by jkryl — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**